MekHQ/src/mekhq/campaign/parts/MissingAeroSensor.java

```diff
@@ -137,4 +137,18 @@ public TechAdvancement getTechAdvancement() {
     public PartRepairType getMRMSOptionType() {
         return PartRepairType.ELECTRONICS;
     }
+
+    // This function is over-ridden from it's parent because Aero Sensors depend on tonnage for small craft, but not for dropships/warships
+    //So we have to use regexes to change the acquistion name if the sensors are for spacecraft
+    @Override
+    public String getAcquisitionName() {
+        if (dropship) {
+            //The below regex splits by the () characters but keeps them in the description
+            String[] sliced = super.getAcquisitionName().split("(?<=\\()|(?=\\))");
+            String descString = sliced[0] + sliced[2] + sliced[3];
```

Comment:
We're trying to move away from abbreviated variable names. You also don't need to include 'String' in the variable name. Simply having this as `description` would likely be a better option.

Reply:
name's been changed to description as requested